mav_msgs/include/mav_msgs/eigen_mav_msgs.h

```diff
@@ -152,7 +169,8 @@ struct EigenTrajectoryPoint {
         jerk_W(Eigen::Vector3d::Zero()),
         snap_W(Eigen::Vector3d::Zero()),
         orientation_W_B(Eigen::Quaterniond::Identity()),
-        angular_velocity_W(Eigen::Vector3d::Zero()){};
+        angular_velocity_W(Eigen::Vector3d::Zero()),
+        angular_acceleration_W(Eigen::Vector3d::Zero()){};
```

Comment:
Do you mind fixing this while your'e at it? Space between ) and {, and no semicolon for function definitions.